stdlib/libblastrampoline_jll/Project.toml

```diff
@@ -0,0 +1,13 @@
+name = "libblastrampoline_jll"
```

Comment:
This shim package seems totally unused?

Reply:
My understanding from @staticfloat was that we do need it. I am not sure how these get used, except that the JLL is what provides the LBT library.

Reply:
https://github.com/JuliaLang/julia/pull/39455#discussion_r577502753

Reply:
https://github.com/JuliaLang/julia/pull/39455/commits/62f86b8d6fbae9fae810635df5de369f8d3876ea